USAGE.md

```diff
@@ -1,370 +1,707 @@
 # Introduction
-This document describes how to query and manipulate ATT&CK data in this repository. Machine-readable ATT&CK data is currently available via two mechanisms:
-* a JSON-based [STIX 2.0](https://oasis-open.github.io/cti-documentation/stix/intro) format.
-* a [TAXII Server](https://medium.com/mitre-attack/att-ck-content-available-in-stix-2-0-via-public-taxii-2-0-server-317e5c41e214) hosting the same STIX 2.0 content
 
-The MediaWiki API was deprecated with the ATT&CK content release in October 2018 and fully removed on February 12th, 2019. Luckily, STIX 2.0 is just JSON and so should be very accessible from Python and other programming languages. If you are using Python, the [python-stix2](https://github.com/oasis-open/cti-python-stix2) library can help you work with the content as shown in the examples below. 
+This document describes how to query and manipulate ATT&CK data in this repository. It is divided into three sections:
```

Comment:
Maybe mention the TAXII server here as well since it's the same content? Unless we have something more specific to TAXII elsewhere.